.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,15 @@
-#### Code Sample, a copy-pastable example if possible
+#### A small, complete example of the issue
```

Comment:
I think it is OK to leave it like this, and it gives a bit structure to the report with those titles (and by using a lot of `#`, the title is not too big)